### PR TITLE
[acl_loader/main.py] handled the priority of ACL ruleprops to be case insensitive

### PR DESCRIPTION
**Pull request by madhanmellanox**
- What I did
I made the priority of ACL rule props in the ACL commands to be case insensitive.

- How I did it
Instead of checking the ACL ruleprops dictionary for "PRIORITY" based key, I made it the actual key to be searched for.

- How to verify it
I verified the fix by testing on my switch using different case sensitive options of keyword "Priority".

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: acl_loader/main.py [hunk 1]
```diff
@@ -676,7 +676,14 @@ def show_rule(self, table_name, rule_id):
         header = ("Table", "Rule", "Priority", "Action", "Match")
 
         def pop_priority(val):
-            priority  = val.pop("PRIORITY")
+            priority = None
+            for key in dict(val):
+                if (key.upper() == "PRIORITY"):
+                    priority  = val.pop(key)
+                    return priority
```
Comment (stepanblyschak) [inline acl_loader/main.py:683]: Can you just break the loop in this point, so lines 684-686 can be removed

**Comment by madhanmellanox** [comment]
> Please, add a UT for acl-loader to cover this

I have added a test case in test_acl.py to cover this already.

**Comment by stepanblyschak** [comment]
> > Please, add a UT for acl-loader to cover this
> 
> I have added a test case in test_acl.py to cover this already.

You don't need VS test case for this fix, take a look at https://github.com/Azure/sonic-utilities/blob/master/sonic-utilities-tests/acl_loader_test.py

File: acl_loader/main.py [hunk 2]
```diff
@@ -713,6 +720,8 @@ def pop_matches(val):
                 continue
 
             priority = pop_priority(val)
+            if priority == None:
```
[the remark this answers is not in the document]
Comment (madhanmellanox) [reply acl_loader/main.py]: Added a test cases in sonic-utilities/sonic-utilities-tests/aclshow_test.py.

**Comment by madhanmellanox** [comment]
This PR is a duplicate PR, this is addressed in https://github.com/Azure/sonic-utilities/pull/918 which got approved and merged. So, closing this PR.

**Pull request closed without merging**